src/components/Flyout/Flyout.tsx

```diff
@@ -164,13 +167,16 @@ export const Flyout: FC<FlyoutProps> = ({
         scrollRef: scrollRef,
         isOpen,
     });
+
     const { buttonProps } = useButton({ onPress: () => toggle() }, triggerRef);
     useEffect(() => {
         const revert = watchModals();
 
         return () => revert();
     }, []);
 
+    useContainScroll(overlayRef, { isDisabled: !isOpen });
```

Comment:
Is there a reason you use an object instead of just a boolean for `isDisabled`?

Reply:
No particular reason other than I try to be consistent with the react-aria hooks, where supplying properties to the hooks is always done as an object for clarity, but it's not essential

Reply:
Fair point, also you immediately know what the argument is used for 👍